src/client/eppo-client.ts

```diff
@@ -144,21 +150,42 @@ export default class EppoClient {
    * @returns a variation value if the subject is part of the experiment sample, otherwise the default value
    * @public
    */
-  public getStringAssignment(
+  public getStringAssignment = (
     flagKey: string,
     subjectKey: string,
     subjectAttributes: Record<string, AttributeType>,
     defaultValue: string,
-  ): string {
-    return (
-      this.getAssignmentVariation(
-        flagKey,
-        subjectKey,
-        subjectAttributes,
-        EppoValue.String(defaultValue),
-        VariationType.STRING,
-      ).stringValue ?? defaultValue
+  ) => this.getStringAssignmentDetails(flagKey, subjectKey, subjectAttributes, defaultValue).value;
```

Comment:
excellent - was thinking how these non-details functions should just call details and then return the value